File: IT/1/Variabili, operatori, strutture di controllo/Esercizi/Java/Esercizio20.java
/* file Esercizio20.java */

public class Esercizio20 {
	public static void main(String [] args) {
		for (int i = 1; i <= 5; i++) {
			for (int j = 1; j <= 5; j++) {
				if (j > i) {
					System.out.print("0" + " ");
				}
				else {
					System.out.print("1" + " ");
				}
			}

			System.out.print(System.lineSeparator());
		}
	}
}

File: IT/1/Variabili, operatori, strutture di controllo/Esercizi/Python/esercizio20.py
# file esercizio20.py

from __future__ import print_function
from sys import stdout
from os import linesep

for i in range(1, 6):
	for j in range(1, 6):
		if j > i:
			stdout.write("0" + " ")
		else:
			stdout.write(str(1) + " ")
	stdout.write(linesep) 

